website/sidebars.json

```diff
@@ -1,7 +1,12 @@
 {
   "docs": {
     "Introduction": ["what-are-uniforms", "motivation", "compare-matrix"],
-    "Getting Started": ["installation", "faq", "migrating-2-to-3"],
+    "Getting Started": [
+      "installation",
+      "faq",
+      "migrating-3-to-4",
+      "migrating-2-to-3"
```

Comment:
Do you think that it's better to order them like this? Both make sense, actually...

Reply:
I chose this order because it puts the v3 to v4 tab at the top, closer to the beginning which may be noticed first if someone was looking for it from top to bottom.